.github/remove_trailing_space.py

```diff
@@ -0,0 +1,11 @@
+# This is used by the pr-description-update action
+# to trim trailing newlines and spaces in pr descriptions
+
+import sys
+
+def remove():
+    line = sys.argv[1]
+    return line.strip()
+
+if __name__ == "__main__":
+    print(remove())
```

Comment:
Kind of sad to have this everywhere. I hope it won't get more complex over time. Otherwise maybe we should consider implementing our own GH action 🤷

Reply:
yeah could make sense to create an update-pr custom action